CONTRIBUTING.md

```diff
@@ -75,6 +73,47 @@ Some companies and organizations are excessively protective with their brands, s
 
 If you are in doubt, feel free to submit it and we'll have a look.
 
+### Assessing Popularity
+
+To be considered popular enough to be within our scope, a brand must be in existence for at least one year (from date of first stable release, where applicable) and meet one of the following metrics of popularity, in order of preference:
+
+1. Its website has a Similarweb global rank of 500k or better.
+    - As Similarweb only update their data once a month, there will be a monitoring window for websites ranked between 450k & 550k until the next update, _unless_ the brand is within scope on any other metric below.
+    - For existing icons in our library, the threshold is dropped from 500k to 750k.
+    - A rank of 2m or worse, without any other metric being provided, will result in the brand being declared outside our scope.
+2. The website's Similarweb rank in any one country is either:
```

Comment:
Note that we are using `1.` for all items of ordered lists. The Markdown processor used by Github converts them to `1.`, `2.`, `3.`...